staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -157,6 +174,10 @@ func handleRootHealthz(checks ...HealthzChecker) http.HandlerFunc {
 				fmt.Fprintf(&verboseOut, "[+]%v ok\n", check.Name())
 			}
 		}
+		if excluded.Len() > 0 {
+			klog.Warningf("cannot exclude some health checks, no health checks are installed matching %v",
```

Comment:
I recommend printing this to the healthz page in addition (if not instead of) the logs.

Reply:
(I expect that the people doing the health checking may not think to look in the logs? Not sure. I guess this is optional :) )

Reply:
Personally I'm more likely to look in the logs, as one anecdata point :-)  I don't think I've ever checked the healthz page directly when diagnosing something (but maybe I should get into that habit!)

Reply:
I'll do both :)